src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -100,6 +92,21 @@
     @Option(name = "-optimize", usage = "boolean switch to optimize index (force merge)")
     public boolean optimize = false;
 
+    @Option(name = "-keepStopwords", usage = "boolean switch to keep stopwords")
+    public boolean keepStopwords = false;
+
+    @Option(name = "-uniqueDocid", usage = "remove duplicated documents with the same doc id when indexing. " +
+      "please note that this option may slow the indexing a lot and if you are sure there is no " +
+      "duplicated document ids in the corpus you shouldn't use this option.")
+    public boolean uniqueDocid = false;
+
+    @Option(name = "-memorybuffer", usage = "memory buffer size")
+    public int memorybufferSize = 2048;
+
+    @Option(name = "-whitelistFile", usage = "when specified only the docids contained in the " +
+      "file will be included in the index, one docid per line.")
```

Comment:
How about just `-whitelist`? "File" is kinda redundant.

Also, start the description with a noun, e.g., "file containing docids, one per line; only specified docids will be indexed."